src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/ExceptionRecordTest.java

```diff
@@ -14,7 +14,7 @@ public void should_upload_blob_and_create_exception_record() throws Exception {
         String zipFileName = ZipFileHelper.randomFileName();
 
         var zipArchive = ZipFileHelper.createZipArchive(
-            singletonList("test-data/exception/1111002.pdf"),
+            singletonList("test-data/1111002.pdf"),
```

Comment:
I think it's better to keep envelopes self contained within a single folder.
We can then refactor this helper so that it takes an entired directory as a single parameter.